repo.go

```diff
@@ -695,6 +696,32 @@ func (r *Repo) AddTargets(paths []string, custom json.RawMessage) error {
 	return r.AddTargetsWithExpires(paths, custom, data.DefaultExpires("targets"))
 }
 
+func (r *Repo) AddDigestTargets(digest string, digestAlg string, length int64, custom json.RawMessage, path string) error {
```

Comment:
The name is a bit poor. Do you mean something like `AddTargetToTargets` (not great either, but you get the idea)?

Reply:
Changed to `AddTargetsWithDigest`, better?